apps/src/applab/GoogleChart.js

```diff
@@ -187,6 +187,20 @@ class PieChart extends GoogleChart {
 }
 GoogleChart.PieChart = PieChart;
 
+class Histogram extends GoogleChart {
+  render_(dataTable, options) {
+    const apiChart = new GoogleChart.lib.visualization.Histogram(
+      this.targetDiv_
+    );
+    apiChart.draw(dataTable, options);
+  }
+
+  verifyData_(data, columns) {
+    super.verifyData_(data, columns, 1);
```

Comment:
Is there an upper limit on the number of columns you can pass to a histogram?

Now that we've got variable enforcement on both the lower and upper bounds of column count, I wonder if we should somehow generalize how we specify that range and pass it into `verifyData_`, or even extract a second helper.